tests/conftest.py

```diff
@@ -52,7 +52,7 @@ async def random_name():
     docker = Docker()
     images = await docker.images.list()
     for img in images:
-        if img["RepoTags"] is None:
+        if img["RepoTags"] in [None, []]:
```

Comment:
This could be simplified as:
```suggestion
        if img["RepoTags"]:
```